Setting the probes to empty dicts breaks the indentation

In openarchiefbeheer, setting these fields in the values.yaml:

```yaml
livenessProbe: {}
readinessProbe: {}
```

results in

```
charts/charts/openarchiefbeheer$ helm lint .
==> Linting .
[INFO] Chart.yaml: icon is recommended
[ERROR] templates/deployment.yaml: unable to parse YAML: error converting YAML to JSON: yaml: line 56: did not find expected key

Error: 1 chart(s) linted, 1 chart(s) failed
```

While setting the probes to:

```yaml
livenessProbe:
  initialDelaySeconds: 60
  periodSeconds: 10
  timeoutSeconds: 5
  failureThreshold: 6
  successThreshold: 1

readinessProbe:
  initialDelaySeconds: 30
  periodSeconds: 10
  timeoutSeconds: 5
  failureThreshold: 6
  successThreshold: 1
```

Results in a chart without errors when running `helm lint`.

We need to find a way to disable the probes without breaking the chart.

--- a/charts/openarchiefbeheer/Chart.yaml
+++ b/charts/openarchiefbeheer/Chart.yaml
@@ -3,7 +3,7 @@ name: openarchiefbeheer
 description: Opstellen, beheren en uitvoeren van vernietigingslijsten, voor gebruik met Zaakgericht werken
 
 type: application
-version: 1.1.0-beta.3
+version: 1.1.0-beta.5
 appVersion: latest
 
 dependencies:

--- a/charts/openarchiefbeheer/templates/deployment.yaml
+++ b/charts/openarchiefbeheer/templates/deployment.yaml
@@ -244,10 +244,16 @@ spec:
             {{- if .Values.extraEnvVars }}
             {{- include "openarchiefbeheer.tplvalues.render" (dict "value" .Values.extraEnvVars "context" $) | nindent 12 }}
             {{- end }}
-            {{- if .Values.worker.livenessProbe }}
+          {{- if .Values.worker.livenessProbe.enabled }}
           livenessProbe:
-            {{- toYaml .Values.worker.livenessProbe | nindent 12 }}
-            {{- end }}
+            exec: 
+            {{- toYaml .Values.worker.livenessProbe.exec | nindent 14 }}
+            initialDelaySeconds: {{ .Values.worker.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.worker.livenessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.worker.livenessProbe.timeoutSeconds }}
+            successThreshold: {{ .Values.worker.livenessProbe.successThreshold }}
+            failureThreshold: {{ .Values.worker.livenessProbe.failureThreshold }}
+          {{- end }}
           resources:
             {{- toYaml .Values.worker.resources | nindent 12 }}
           command:

--- a/charts/openarchiefbeheer/values.yaml
+++ b/charts/openarchiefbeheer/values.yaml
@@ -257,6 +257,11 @@ worker:
   resources: {}
   livenessProbe: 
     enabled: false
+    initialDelaySeconds: 60
+    periodSeconds: 10
+    timeoutSeconds: 10
+    failureThreshold: 3
+    successThreshold: 1
   autoscaling:
     enabled: false
     minReplicas: 1

--- a/charts/openarchiefbeheer/values.yaml
+++ b/charts/openarchiefbeheer/values.yaml
@@ -255,8 +255,8 @@ worker:
   concurrency: 4
   podLabels: {}
   resources: {}
-  livenessProbe: {}
-  readinessProbe: {}
+  livenessProbe: 
+    enabled: false
   autoscaling:
     enabled: false
     minReplicas: 1

